src/adcm_pytest_plugin/steps/common.py

```diff
@@ -38,24 +39,26 @@ def assume_step(title, exception=None):
         raise ValueError("This is expected exception")
 
     """
-    if callable(title):
+    if callable(title):  # pylint: disable=no-else-return
         return AssumeStepContext(title.__name__, exception)(title)
     else:
         return AssumeStepContext(title, exception)
 
 
 class AssumeStepContext:
+    """Step wrapper like `allure.step`"""
```

Comment:
I think this docstring should be more detailed

Reply:
fix